ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -314,10 +314,14 @@ class RootViewController: UIViewController, UITableViewDataSource, UITableViewDe
 
     @objc private func showSelectionMode() {
         isInSelectionMode = true
+        msfNavigationController?.contractNavigationBar(animated: false)
```

Comment:
So when we dismiss selection mode we are animated, but when we show selection mode we don't show an animation? I know it's demo code so probably doesn't matter.

Reply:
I was just playing around with it but I could make it consistent.